pipdeptree.py

```diff
@@ -418,6 +466,12 @@ def main():
                             '"raw" output that may be used by external tools. '
                             'This option overrides all other options.'
                         ))
+    parser.add_argument('--output', dest='output_format', required=False,
```

Comment:
Using choices there would be awesome, but would mean to make `graphviz` a mandatory dependency as it'd be required for argument parsing in any case. I can do that, but I'm not sure if it's desired.